### ISSUE TITLE
New group fields in 4.3

### ISSUE THREAD

**Issue by emmarichardson** [emmarichardson/local_autogroup#50]
Getting the following error in 4.3...Fields list in snapshot record does not match fields list in 'groups'. Record is missing fields: visibility, participation.
4.3 has added two new fields to the groups table.  I have tried adding the missing fields to both domain/group and autogroup_set but error persists.  It does not seem to affect the functionality.
@ak4t0sh do you have time to see if you can come up with a fix?

**Comment by ak4t0sh** [comment]
I do not think I'll have time to work on it soon but if a PR is done I think I could check on it.
FYI this field has been introduce in 4.2 : https://github.com/moodle/moodle/blob/main/group/upgrade.txt

**Comment by emmarichardson** [comment]
Yeah, I skipped 4.2 altogether - still haven't upgraded to 4.3 but getting close to thinking about it!  Thanks for letting me know.
